packages/calcite-components/src/components/action/action.tsx

```diff
@@ -42,7 +42,7 @@ import { CSS, SLOTS } from "./resources";
 
 /**
  * @slot - A slot for adding a `calcite-icon`.
- * @slot tooltip - A slot for adding a `calcite-tooltip`.
+ * @slot tooltip - [Deprecated] use `id` instead. A reference to a `calcite-tooltip`.
```

Comment:
The deprecation message should suggest using a tooltip instead. +@geospatialem @DitwanP for doc guidance.

Reply:
WDYT about the following for folks? cc: @DitwanP

 ```suggestion
 * @slot tooltip - [Deprecated] Use the `calcite-tooltip` component instead.
```